specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json

```diff
@@ -375,6 +383,31 @@
         }
       }
     },
+    "policyDetailSettings": {
```

Comment:
Please start definition's name with capital letter for consistency

Reply:
Sure. I will address the same.